libs/shared/utils/src/lib/constants/currency-dictionaries.ts

```diff
@@ -0,0 +1,17 @@
+export const currencyDictionary: Record<string, string> = {
```

Comment:
Could you add the same name between `const` and file or vice versa?, that would be more coherent I think